packages/performance/src/services/perf_logger.ts

```diff
@@ -83,18 +82,18 @@ interface TraceMetric {
   counters?: Array<{ key: string; value: number }>;
   custom_attributes?: Array<{ key: string; value: string }>;
 }
+
 /* eslint-enble camelcase */
 
-let logger: Logger | undefined;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
The eslint suppression doesn't seem necessary.

Reply:
Done.